src/api/middleware/validators/validatorUtils.js

```diff
@@ -55,9 +52,17 @@ const isObjectId = (id) => {
     return true;
 };
 
-const offerLimitNotReached = async (owner, publishDate, publishEndDate, offer) => {
-    const concurrentOffers = await (new CompanyService()).
-        getOffersInTimePeriod(owner, publishDate, publishEndDate, offer);
+/**
+ * Checks if the concurrent offers of a given owner have not exceeded the defined limit
+ * @param {*} OfferModel Either the default Offer model or its constructor (if cast inside the model)
+ * @param {*} owner Owner of the offer
+ * @param {*} publishDate Publish date of the
+ * @param {*} publishEndDate Date in which the offer will end
+ */
+const concurrentOffersNotExceeded = (OfferModel) => async (owner, publishDate, publishEndDate) => {
+    // We need to pass an offer instance because Offer.find is not callable inside the offer model
```

Comment:
You're not passing an Offer instance, you're passing the class itself. In JS, you can do that by importing it and using it as a variable, or if you are inside an instance of that class, it is accessible in the `constructor` field, hence the `this.constructor` passing in the model. Update the comments.

Reply:
You're right, I forgot to change that in the last commit. I'll change it and check if I left some other comments like that

Reply:
I fixed another comment besides this one. I hope that settles everything